bayesian/utility/v2/component_analyses.py

```diff
@@ -241,26 +290,29 @@ def get_known_unknown_pkgs(
     return stack_recommendation, unknown_pkgs
 
 
-def get_known_clair_pkgs(
-        ecosystem: str, graph_response: Dict) -> Tuple[List[Dict], Set[Package]]:
+def check_vulnerable_package(version, version_string):
+    """Check if the request version available in vulnerable_versions."""
+    if(version in version_string.split(",")):
+        return True
+    return False
+
+
+def get_known_pkgs(graph_response: Dict, packages_list: Dict) -> List[Dict]:
     """Analyse Known and Unknown Packages."""
-    stack_recommendation = []
+    for temp in packages_list:
```

Comment:
this is the best place to create the temp map. you are anyways running a loop and traversing through the input

Reply:
done